src/components/Review/ReviewForm.tsx

```diff
@@ -6,7 +6,35 @@
   - 내에서 자유롭게 컴포넌트 분리해도 되고 도입하고 싶은 거 있으면 언제든지 환영 ~~
   - 화이팅텡!
 */
+import React, { useState } from "react";
+import styled from "styled-components";
+
+import PhotoAttach from "./PhotoAttach";
 
 export default function ReviewForm() {
-  return <div>ReviewForm</div>;
+  const [imgFile, setImgFile] = useState<File>();
+  const handleImgFile = (selectImgFile: File) => {
+    setImgFile(selectImgFile);
+  };
+  // 파일 업로드 함수
+  const uploadFile = function (e: React.MouseEvent<HTMLSpanElement, MouseEvent>) {
+    const formData = new FormData();
+    if (imgFile) {
+      formData.append("uploadImage", imgFile, imgFile.name);
+      // const config = {
+      //   Header: {
+      //     "content-type": "multipart/form-data",
+      //   },
+      // };
+      // axios.post('', formData, config);
+    }
+  };
+  return (
+    <StPhotoAttachWrapper>
+      <PhotoAttach imgFile={imgFile} handleImgFile={handleImgFile} />
+    </StPhotoAttachWrapper>
+  );
 }
+const StPhotoAttachWrapper = styled.div`
```

Comment:
```suggestion
const StPhotoAttachWrapper = styled.section`
```

시멘틱 태그를 활용해도 좋을 것 같아 !!